### DISCUSSION TITLE
minify css and js

### DISCUSSION THREAD

**Discussion by alireza-hnzpv** [livewire/livewire#8626] [7. Bugs (archived)] [answered]
### Livewire version

3.4

### Laravel version

10.48.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

I am testing my site on gtmetrix. One of the negative points that he gave me was the minification of css and js, which was getting stuck in the livewire file. Please minify it

### Repository or Wirebox URL

none

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by PhiloNL** [comment, marked as answer]
If you set `APP_DEBUG=false` it should use a minified version.
